### DISCUSSION TITLE
talker commands echoing with newer tt++ versions

### DISCUSSION THREAD

**Discussion by pen-helm** [scandum/tintin#59] [Bugs]
When I use tt++ 2.02.05 or 2.02.12b to log into my talker, there is a glitch.  I log in with the below script, and when I type :commands to the talker, they echo.  This did not happen with tt++ 2.0.8.  It does not happen if I edit the script to not enter the password either (and type it manually.)

To reproduce, just use the below script and then type :tests [return].  It will either display 
 You emote : temp tests    [this is what we want]
or
:tests You emote : temp tests   [not what we want.]

#config verbatim on
#config {command echo} off
#session Brigandine brigandine.org 9390
#tick {keepalive} {#send {\xFF\xFB\x1F\} {600} }
temp
password1
#delay {3} {#echo test}

#delay {3} {#SPLIT}

I tried adding a \n or a \r after "password1", just as a guess, but it didn't help.

**Comment by scandum** [comment]
As far as I can tell it's a problem with the talker, and not with tintin.

Using `#delay 1 password1` seems to be a workaround.

Another workaround seems to be adding
```
#event {IAC WONT ECHO} {#send {\xFF\xFE\x01\}}
```
Which adds the echo confirmation tintin used to send, but there were a couple of muds managing to turn that response into an infinite loop, so I ended up disabling it.

**Reply by pen-helm** [reply]
Problem solved!  Thank you!
